docs/examples/tutorial/fib1/fib.pyx

```diff
@@ -2,5 +2,5 @@ def fib(n):
     """Print the Fibonacci series up to n."""
     a, b = 0, 1
     while b < n:
-        print b,
+        print(b)
```

Comment:
That's not the same, though, since it will add line breaks after each number.
But I'm not sure if I want to see the complexity of a `__future__` import in this example ...

Reply:
I changed it all the way to use new style printing.

Reply:
Thanks for correcting that, I'll be more careful next time.